.github/testing.md

```diff
@@ -0,0 +1,78 @@
+### Development Environment
+This repository targets PHP 8.1 and Node JS 20.x.
```

Comment:
```suggestion
This repository targets PHP 8.2 and Node JS 20.x.
```

Reply:
I'd probably link to our composer.json and package.json files here otherwise this will be perpetually out of date.